LibreNMS/OS/Timos.php

```diff
@@ -926,8 +926,10 @@ public function discoverEntityPhysical(): Collection
     private function parseIpField(array $data, string $ngField): ?string
     {
         if (isset($data[$ngField])) {
+            $raw = $data[$ngField];
+
             try {
-                return IP::parse($data[$ngField])->uncompressed();
```

Comment:
oops, you removed your if statement.  Now it will try to parse all of them as hex strings.

Reply:
Like this :)